src/dist/component/mod.rs

```diff
@@ -10,7 +10,7 @@ mod transaction;
 // The representation of a package, its components, and installation
 mod package;
 // The representation of *installed* components, and uninstallation
-mod components;
+pub mod components;
```

Comment:
I believe this change is not necessary since it is placed right after `pub use components::*` :)